pkgs/data/fonts/xkcd-font/default.nix

```diff
@@ -11,11 +11,22 @@ in fetchFromGitHub {
   rev = "5632fde618845dba5c22f14adc7b52bf6c52d46d";
 
   postFetch = ''
-    tar xf $downloadedFile --strip=1
+    pushd $out
+
     install -Dm444 -t $out/share/fonts/opentype/ xkcd/build/xkcd.otf
     install -Dm444 -t $out/share/fonts/truetype/ xkcd-script/font/xkcd-script.ttf
+
+    # remove unrelated files
```

Comment:
I came up with a shorter way of doing this https://github.com/NixOS/nixpkgs/pull/175381/files#diff-065071edaa386151863b7c71f585da7d8d1d15f9bd21a65a9f1cbb4ccdcde26aR13

Reply:
@Artturin If you don't mind, could you include it in your PR?